src/generateDoubleFunction.test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const generateDoubleFunction = require('./generateDoubleFunction');
+
+test('First call returns function', () => {
+  expect(generateDoubleFunction((x) => x))
+    .toBeInstanceOf(Function);
+});
+
+test('Second call returns doubled value', () => {
```

Comment:
suggestion: Calling generateDoubleFunction result returns doubled result of the initial function.

Reply:
generateDoubleFunction(f)(x) === 2 * f(x)